spring-core/src/test/java/org/springframework/core/serializer/SerializerTests.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2002-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.core.serializer;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.core.serializer.support.SerializationDelegate;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+class SerializerTests {
+
+	private static final String SPRING_FRAMEWORK = "Spring Framework";
+
+
+	class SpyStringSerializer<T> implements Serializer<T> {
```

Comment:
These helper classes should be `static` and placed at the end of the test class, after test methods.

Reply:
Done